client/styles/sc-text-paragraph-num-styles.html

```diff
@@ -2,437 +2,453 @@
         content: ""
       }
 
-      .sen:before {
+      .textual-info-paragraph.sen:before {
         content: "Sen "
       }
 
-      .sht:before {
+      .textual-info-paragraph.sht:before {
         content: "SHT "
       }
 
-      .svsv:before, .svv:before {
+      .textual-info-paragraph.svsv:before, .svv:before {
         content: "SNP "
       }
 
-      .ss:before {
+      .textual-info-paragraph.ss:before {
         content: "SS "
       }
 
-      .sv:before {
+      .textual-info-paragraph.sv:before {
         content: "SV "
       }
 
-      .t:before, .t-linehead:before {
+      .textual-info-paragraph.t:before, .t-linehead:before {
         content: "T "
       }
 
-      .tb:before {
+      .textual-info-paragraph.tb:before {
         content: "TB "
       }
 
-      .thv:before {
+      .textual-info-paragraph.thv:before {
         content: "THV "
       }
 
-      .tri62:before, .tri95:before {
+      .textual-info-paragraph.tri62:before, .tri95:before {
         content: "Tri "
       }
 
-      .us:before, .uvs:before {
+      .textual-info-paragraph.us:before, .uvs:before {
         content: "Ud "
       }
 
-      .ttc:before {
+      .textual-info-paragraph.ttc:before {
         content: "TTC "
       }
 
       /*a.tu:before {content:" "}*/
-      .uv:before {
+      .textual-info-paragraph.uv:before {
         content: "Uv "
       }
 
-      .vgns:before {
+      .textual-info-paragraph.vgns:before {
         content: "Vagga/verse "
       }
 
-      .vai58:before, .vai59:before, .vai61:before {
+      .textual-info-paragraph.vai58:before, .textual-info-paragraph.vai59:before, .textual-info-paragraph.vai61:before {
         content: "Vai "
       }
 
-      .var-v:before {
+      .textual-info-paragraph.var-v:before {
         content: "Var-v "
       }
 
-      .vns:before {
+      .textual-info-paragraph.vns:before {
         content: "SC verse "
       }
 
-      .vnp:before {
+      .textual-info-paragraph.vnp:before {
         content: "PTS verse "
       }
 
-      .vn:before {
+      .textual-info-paragraph.vn:before {
         content: "Vn "
       }
 
-      .wal48:before, .wal50:before, .wal52:before, .wal57c:before, .wal58:before, .wal59a:before, .wal60:before, .wal61:before, .wal68a:before, .wal70a:before, .wal70b:before, .wal76:before, .wal78:before, .wal80c:before {
+      .textual-info-paragraph.wal48:before, .textual-info-paragraph.wal50:before, .textual-info-paragraph.wal52:before,
+      .textual-info-paragraph.wal57c:before, .textual-info-paragraph.wal58:before, .textual-info-paragraph.wal59a:before,
+      .textual-info-paragraph.wal60:before, .textual-info-paragraph.wal61:before, .textual-info-paragraph.wal68a:before,
+      .textual-info-paragraph.wal70a:before, .textual-info-paragraph.wal70b:before, .textual-info-paragraph.wal76:before,
+      .textual-info-paragraph.wal78:before, .textual-info-paragraph.wal80c:before {
```

Comment:
hmm, maybe `.textual-info-paragraph[class^="wal"]:before` would work here? or if not, maybe `[class^="textual-info-paragraph wal"]:before`?

Reply:
Doubt it if that would work, they are very different classes.